### PR TITLE
V2.6.4 -- include harvester for rke2 templates (#4)

### PR DESCRIPTION
**Pull request by slickwarren**
* update to allow selection for Harvester (questions.yaml, config.yaml, example values.yaml)
* update questions.yaml to read from first item in `nodepools` list
* update default k8s version list
* updated packaged v0.0.1 template
* various whitespace fixes

### CODE REVIEW

**Review by jakefhyde — APPROVED** [review]

File: charts/questions.yaml [hunk 1]
```diff
@@ -67,113 +69,113 @@ questions:
...
   - variable: monitoring.values
-    default: 
+    default:
     label: "Monitoring Values"
     description: "Custom monitoring chart values"
     type: multiline
     group: "Monitoring"
-- variable: nodepool.name
-  default: 
+- variable: nodepools.0.name
```
Comment (thedadams) [inline charts/questions.yaml:83]: Is there a way to loop through the nodepools instead of only using the first one?

**Review by thedadams — APPROVED** [review]
With my limited understanding of cluster templates, this looks fine. Only one question.

**Review by jiaqiluo — APPROVED** [review]
LGTM. 
We still need a better way to handle the questions for multiple `nodepool`s

File: charts/questions.yaml (hunk 1 above)
Comment (slickwarren) [reply charts/questions.yaml:83]: When Daishan was working on this, the answer he gave me was no, its not possible to loop through for the questions.yaml . I was talking with @bashofmann about this solution and if my understanding is correct this is just a limitation of the tool at this time.

**Review by kinarashah — APPROVED** [review]

**Pull request merged**